app/views/miq_request/_request_dialog_details.html.haml

```diff
@@ -0,0 +1,36 @@
+.form-group{:id => "field_#{field.id}_tr", :style => field.visible ? 'display:block' : 'display:none'}
+  %label.control-label.col-md-2{:title => field.description}
+    = field.label
+  .col-md-8{:title => field.description}
+    - case field.type
+    - when 'DialogFieldTextBox', 'DialogFieldTextAreaBox'
+      - if field.protected?
+        ********
+      - else
+        = h(field.value)
+
+    - when 'DialogFieldCheckBox'
+      = check_box_tag(field.name, "1", field.checked?, {:disabled => !edit})
```

Comment:
@eclarizio where is `edit` getting set, i don't see that being passed in either.

Reply:
Ah, that we don't need since it's always going to be not edit. I'll change it to be hardcoded to `true`. Good catch 👍